### PR TITLE
[DM-28131] Initial version

### PR DESCRIPTION
**Pull request by rra**

### CODE REVIEW

**Review by athornton — APPROVED** [review]
All this seems quite sensible.

File: index.rst [hunk 1]
```diff
@@ -1,61 +1,134 @@
...
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Abstract
+========
 
-  Use the following syntax for sections:
+The Interim Data Facility (IDF) is hosted at Google using Google Cloud Platform (GCP).
+Administrators of the IDF must be able to authenticate to the Google infrastructure to debug problems and must have a mechanism to make changes to the IDF configuration.
+This document outlines the identity, authentication, and authorization model used for the IDF and discusses the security goals that it is intended to address.
+It also discusses some non-default configuration options for the GCP organization that are set for security reasons.
```
Comment (frossie) [inline index.rst:12]:
```suggestion
Note that this document specifically deals with authentication to the Google infrastructure, which by policy is limited to Rubin Operations staff that are developing and supporting services deployed on the IDF. It does not pertain to users of these services from our scientific community, who do not have infrastructure accounts, but authenticate through our service layer. 
```

**Review by frossie — APPROVED** [review]
LGTM

**Review by dspeck1 — APPROVED** [review]
Looks great!  Aaron and I both reviewed and no changes.

**Pull request merged**